_v2_1/0_sende_post.md

```diff
@@ -9,17 +9,18 @@ For å sende post så må du først lage en `DigitalPost` eller en `FysiskPost`,
 ### Opprette digital post
 
 ```java
-Sertifikat mottakerSertifikat = null; //Mottakers sertifikat fra Oppslagstjenesten
-String orgnrPostkasse = "123456789";
+Sertifikat mottakerSertifikat = null;   //Fås fra Oppslagstjenesten
+String orgnrPostkasse = null;           //Fås fra Oppslagstjenesten
 
 Mottaker mottaker = Mottaker.builder("99999999999",
         "ola.nordmann#2222", mottakerSertifikat, orgnrPostkasse).build();
```

Comment:
Det jeg egentlig mente med https://github.com/difi/sikker-digital-post-klient-java/pull/46#discussion_r67307980 var at postkasseadressen (altså «ola.nordmann#2222») også kommer fra Oppslagstjenesten og det bør tydeliggjøres på samme måte som for `mottakerSertifikat` og `orgnrPostkasse` :)